apps/src/applab/applab.js

```diff
@@ -761,90 +739,118 @@ Applab.init = function(config) {
   AppStorage.populateTable(level.dataTables, false); // overwrite = false
   AppStorage.populateKeyValue(level.dataProperties, false); // overwrite = false
 
-  studioApp.init(config);
-
-  var viz = document.getElementById('visualization');
-  var vizCol = document.getElementById('visualizationColumn');
-
-  if (!config.noPadding) {
-    viz.className += " with_padding";
-    vizCol.className += " with_padding";
-  }
-
-  if (config.embed || config.hideSource) {
-    // no responsive styles active in embed or hideSource mode, so set sizes:
-    viz.style.width = Applab.appWidth + 'px';
-    viz.style.height = (shouldRenderFooter() ? Applab.appHeight : Applab.footerlessAppHeight) + 'px';
-    // Use offsetWidth of viz so we can include any possible border width:
-    vizCol.style.maxWidth = viz.offsetWidth + 'px';
-  }
+  React.render(React.createElement(AppView, {
```

Comment:
I think this is a great way of transitioning from our current world to our new world. However, long term I think we want to discourage having this much content in our createElement call (to some extent referring to number of properties, but mostly referring to having full function definitions).

If it's not difficult, I'd love to get to the place in the nearish future where we can have JSX instead of createElement which will discourage huge objects like this. 

In any case, no changes are needed in this regards for this PR